testing/openttd/APKBUILD

```diff
@@ -0,0 +1,49 @@
+# Contributor: Adrian Siekierka <kontakt@asie.pl>
+# Maintainer: Adrian Siekierka <kontakt@asie.pl>
+pkgname=openttd
+pkgver=1.6.1
+pkgrel=0
+pkgdesc="Open source simulation game based upon Transport Tycoon Deluxe"
+url="http://openttd.org/"
+arch="all"
+license="GPL2"
+depends=""
+makedepends="libpng-dev zlib-dev sdl-dev lzo-dev xz-dev freetype-dev fontconfig-dev"
+subpackages="$pkgname-doc $pkgname-lang"
+source="https://binaries.openttd.org/releases/1.6.1/openttd-1.6.1-source.tar.xz"
+builddir="$srcdir/$pkgname-$pkgver"
+
+build() {
+	cd "$builddir"
+
+	# OpenTTD's builtin strip does not work when cross-compiling,
+	# but it will be done automatically later in the package build.
+	./configure \
+		--build=$CBUILD \
+		--host=$CHOST \
+		--disable-strip \
+		--prefix-dir=/usr \
+		--binary-dir=bin \
+		--install-dir="${pkgdir}" \
+		--with-sdl --with-zlib --with-liblzma --with-liblzo2 \
+		--with-freetype --with-fontconfig --without-icu \
+		|| return 1
+	make || return 1
+}
+
+package() {
+	cd "$builddir"
+	make install || return 1
+}
+
+lang() {
```

Comment:
Please declare variable `pkgdesc` inside this function with description of the subpackage (e.g. `pkgdesc="$pkgdesc (localizations)"`).